pages/Seismic-Axe.md

```diff
@@ -0,0 +1,2 @@
+The Seismic Axe is a [Weapon](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Weapons) added in Slimefun.<br>
+It is an Iron Axe with a special ability: when used (right click), it launches up to 10x1 blocks up in the air in the direction the Player is looking at, and pushes any entity in its way 4 blocks away. However, this consumes 4 points of durability, so using this ability can deplete the item very quickly.
```

Comment:
It doesn't actually launch blocks up, it is only a visual effect, so make sure to mention that.
Otherwise people may assume it can be used to move blocks.